assignment/index.html

```diff
@@ -62,19 +62,42 @@
 
       ===================== */
 
-      var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
-
-      /* =====================
-
-      End code
-
-      ===================== */
-
-
-      jsonToCsv(healthCenters);
+      let new_array=[['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']]
+        for (var i=1; i<healthCenters.length; i++) {
```

Comment:
It'd be good to not have to type out each of these keys manually.

Take a look how at this: `Object.keys(healthCenters[0])` Does all the work for you!